### ISSUE TITLE
Everyone stays muted

### ISSUE THREAD

**Issue by Purga99** [michail-nikolaev/task-force-arma-3-radio#1244]
**Arma 3 Version:** `1.68` (most updated, recent one) (stable / rc / dev)
**CBA Version:** `3.2.1` (stable / dev + commit hash)
**Task Force Radio Version:** `1.-1.0.235` (stable / dev + commit hash)

**Mods:**
- `@CBA_A3`
- `@task_force_radio

**Description:**
- Whenever I get off arma and stop using TFR, all the people that I had local muted by TFR in the game stays locally unmuted when I leave the game. This never used to happen.

**Steps to reproduce:**
- Use TFR
- Go in-game
- Leave in-game
- Stop using TFR

**Where did the issue occur?**
- Dedicated Multiplayer

**RPT log file:**
- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).

**Comment by dedmen** [comment]
`stays locally unmuted when I leave the game`
But.. What you are describing is correct behaviour. Except that's a typo. If so.. Cannot reproduce and hundreds of other users also can't.

**Comment by Purga99** [comment]
@dedmen No, people who you aren't interacting with or you can't physically hear in-game are muted when you have the plugin enabled but what used to happen was as soon as you were done using tfr and done playing arma, you'd leave the game and everyone on teamspeak in TFR would go unmuted.

**Comment by dedmen** [comment]
That is still happening now. With the difference of it only unmuting People that were muted by TFAR. so People you had locally muted that didn't play with you will stay muted.

**Comment by Purga99** [comment]
@dedmen Any way to fix it on your guys end?

**Comment by dedmen** [comment]
As I said. I can't reproduce it. And hundreds of other Player don't have that issue. I don't know why that happens. When the Game disconnects from Teamspeak then Teamspeak unmutes everyone.
Do you hear the TFAR Sound when the Game ends? It should make a disconnect sound and that's exactly when it unmutes everyone.

**Comment by nomisum** [comment]
I can confirm having to unmute some players after the game manually or even having to mute/unmute to hear them again. But this is not easily reproducable and maybe every 4th or 5th gaming session for one guy out of 20. Not sure if this is even the same bug as described here and probably hell to debug.

**Comment by Purga99** [comment]
@dedmen Yeah I hear the TFAR sound when the game ends. Like 80% get unmuted and the rest stay muted.

**Comment by dedmen** [comment]
TODO: On game End unmute everyone in current channel before moving out when in serious mode

This might fix it. Or atleast hide the symptoms
